### PR TITLE
Updated and Improved README.md

### PR DESCRIPTION
**Pull request by surajphulara**
Corrected spelling mistakes and grammatical errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,19 @@
 
 This is the code for https://deno.land/
 
-This website consists of two parts
+This website consists of two parts :
```
Comment (kitsonk) [inline README.md:7]:
```suggestion
This website consists of two parts:
```

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ and https://github.com/denoland/registry and have faster deployment.
 This is written in React / TailwindCSS / Vercel / CloudFlare Workers. Not in
 Deno. Ideally this could be ported to Deno at some point but we are in need of a
 new website and dogfooding takes too long. We hope to see this code ported to
-Deno with minimal developer flow interrupted (in particular, we need the ability
+Deno with minimal inerruption in developer flow  (in particular, we need the ability
```
Comment (kitsonk) [inline README.md:32]:
```suggestion
Deno with minimal interruption in developer flow (in particular, we need the ability
```

**Review by kitsonk — CHANGES_REQUESTED** [review]

**Comment by kitsonk** [comment]
You also need to get CI to pass before we will consider this PR (basically it is failing because you are not running the linter).

**Comment by surajphulara** [comment]
> You also need to get CI to pass before we will consider this PR (basically it is failing because you are not running the linter).
please guide me how to do that

**Comment by lucacasonato** [comment]
Clone the repository and run `yarn format`

**Comment by lucacasonato** [comment]
Closing due to inactivity.

**Pull request closed without merging**